.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+
+updates:
+
+  # Maintain dependencies for GitHub Actions
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: weekly
+
+  # Maintain dependencies for python
+- package-ecosystem: pip
+  directory: /
+  schedule:
+    interval: daily  # TODO s/daily/weekly once dependabot has settled
```

Comment:
It's good to start out with daily for the time being, while making sure that dependabot is working as expected.

I had been leaning towards going with monthly in the long-term, but the more I think about it, weekly would probably be best.  In particular, having this be monthly would increase the chances of us having to deal with more than one thing breaking in a single PR.  And too often in my experience...doubling the problems that we have to deal with more than doubles the effort needed to fix the combined problems.